lib/matplotlib/tests/test_ticker.py

```diff
@@ -733,7 +748,8 @@ def test_params(self, input, expected):
 
         UNIT = 's'  # seconds
         DIGITS = '0123456789'  # %timeit showed 10-20% faster search than set
-
+        plt.rcdefaults()
```

Comment:
this line is not needed (in fact it should not be used because the test rcdefaults are slightly different from the "real" rcdefaults (set by rcdefaults()) for backcompat reasons).

Reply:
Thanks! didn't know that. removed it.